west.yml

```diff
@@ -277,7 +277,7 @@ manifest:
       revision: 2b76c641749725ac90c6ac7959ca7718804cf356
       path: modules/lib/gui/lvgl
     - name: mbedtls
-      revision: 66ed2279d6222056af172c188eaf4dcfed481032
+      revision: pull/52/head
```

Comment:
Module PR was merged, please update to the new SHA

Reply:
\o/ done